.github/workflows/release.yaml

```diff
@@ -10,20 +10,20 @@ jobs:
     runs-on: ubuntu-latest
     name: goreleaser
     steps:
-    - name: Set up Go 1.13
-      uses: actions/setup-go@v1
-      with:
-        go-version: 1.13
-      id: go
-    - name: Check out code into the Go module directory
-      uses: actions/checkout@v1
-    - name: Install package dependencies
-      run: |
-        echo "::set-env name=GO_VERSION::$(go version | awk '{ print $3}' | sed 's/^go//')"
-        make get
-    - name: Release via goreleaser
-      uses: goreleaser/goreleaser-action@master
-      with:
-        args: release --rm-dist --release-notes .release-notes.md
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+      - name: Install package dependencies
+        run: |
+          echo "::set-env name=GO_VERSION::$(go version | awk '{ print $3}' | sed 's/^go//')"
+          make get
+      - name: Release via goreleaser
+        uses: goreleaser/goreleaser-action@master
+        with:
+          args: release --rm-dist --release-notes .release-notes.md
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN_PERSONAL }}
```

Comment:
shouldn't you revert this before merging?

Reply:
This change is needed. See https://help.github.com/en/actions/automating-your-workflow-with-github-actions/authenticating-with-the-github_token#permissions-for-the-github_token

Reply:
Ok cool, but can we call it something other than GITHUB_TOKEN_PERSONAL?

I would prefer `GORELEASER_GITHUB_TOKEN` as it obscures that this is a personal api key

Reply:
This token is not goreleaser specific. Mayberry personal can be something else but anybody who knows something about actions can tell it is an API token.

Reply:
alright, yea, just pick a different name than Personal. I have hesitation with using personal on a public project.

Reply:
Here’s an interesting take on the topic https://stackoverflow.com/questions/533965/why-is-security-through-obscurity-a-bad-idea

Reply:
@gotwarlost Got any thoughts on this?

Reply:
How about GITHUB_ACCESS_TOKEN?

Reply:
What about `QBEC_GITHUB_TOKEN` 🤷‍♂

Reply:
sounds good!

Reply:
It is not tied to qbec. It’s a personal token that’ll have access to all repos as the account.

Reply:
I deleted my comment when I saw yours :)